lib/zendesk_api/middleware/response/sanitize_response.rb

```diff
@@ -0,0 +1,21 @@
+module ZendeskAPI
+  module Middleware
+    module Response
+      class SanitizeResponse < Faraday::Response::Middleware
+        def on_complete(env)
+          to_utf8!(env[:body])
+        end
+
+        def to_utf8!(str)
+          str.force_encoding(Encoding::UTF_8)
```

Comment:
The less we force the encoding, the better. We should really leave it up to ruby to figure out the default encoding.